View Assist device configuration/device_config_example.yaml

```diff
@@ -39,4 +39,4 @@ template:
         alarm: "idle"
         cycle_view: ""
         do_not_disturb: false
-        
+        
```

Comment:
Should consider removing this commit from the PR as it appears to be an extraneous line change